src/middlewares.js

```diff
@@ -454,3 +469,8 @@ function invalidRequest(req, res) {
   res.status(403);
   res.end('{"error":"unauthorized"}');
 }
+
+function malformedContext(req, res) {
+  res.status(500);
```

Comment:
This error is caused by a developer mistake, so it should be a 4xx response code.

I suggest `400 Bad Request`:
> The request could not be understood by the server due to malformed syntax. The client SHOULD NOT repeat the request without modifications.